common-libs/common/package.json

```diff
@@ -49,6 +49,7 @@
     "@affinidi/tools-common": "^1.2.0",
     "@affinidi/url-resolver": "^2.0.0",
     "@affinidi/vc-common": "^1.6.0",
+    "@types/sinon-chai": "^3.2.8",
```

Comment:
please move dev deps to dev deps (e.g. all types are devDeps, all deps for testing are devDeps etc